cmake/projects/Boost/schemes/url_sha1_boost_library.cmake.in

```diff
@@ -251,6 +251,12 @@ set(
     "--user-config=${boost_user_jam}"
     --with-@HUNTER_PACKAGE_COMPONENT@
 )
+if("@MINGW@")
+  if("@CMAKE_CROSSCOMPILING@")
+     # cross compile from "linux" to "windows" using mingw
+     set(build_opts target-os=windows ${build_opts})
```

Comment:
I don't know if this works, but I'll remove the condition and let the CI buildserver test it